src/routers/login/controller.spec.ts

```diff
@@ -0,0 +1,144 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { LoginController } from './controller';
+import { Model } from 'mongoose';
+import { IUserModel, IUser } from "../../models/User";
+import { Request, Response } from 'express';
+import * as models from '../../models';
+
+describe('LoginController test', () => {
+    var UserMock = function () {};
+    beforeEach(() => {
+        UserMock.prototype.findOne = function () {
```

Comment:
Nice one, If sinon does not working with UserMock, I like this way, it's very cleaner